doc/source/tune/examples/tune_analyze_results.ipynb

```diff
@@ -0,0 +1,912 @@
+    "## Trial-level Analysis: Working with an individual `Result`"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "ae8b394d-fedc-4522-990a-2cf50fbf4276",
+   "metadata": {},
+   "source": [
+    "Let's take a look at the result that ended with the best `mean_accuracy` metric. By default, `get_best_result` will use the same metric and mode as defined in the `TuneConfig` above. However, it's also possible to specify a new metric/order in which results should be ranked."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "id": "d7e0ce5b-e278-40d1-aee6-e2840a9fa77f",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from ray.air import Result\n",
+    "\n",
+    "# Get the result with the maximum test set `mean_accuracy`\n",
+    "best_result: Result = result_grid.get_best_result()\n",
+    "\n",
+    "# Get the result with the minimum `mean_accuracy`\n",
+    "worst_performing_result: Result = result_grid.get_best_result(\n",
+    "    metric=\"mean_accuracy\", mode=\"min\"\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "58045cd9-3894-48b8-8196-1c81a00feb7d",
+   "metadata": {},
+   "source": [
+    "We can examine a few of the properties of the best `Result`. See the [API reference](result-docstring) for a list of all accessible properties."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 9,
+   "id": "1f1c3d9b-ec32-439d-9e8e-e9afb58f1331",
+   "metadata": {},
+   "outputs": [
+    {
+     "data": {
+      "text/plain": [
+       "{'lr': 0.0876741358605454, 'momentum': 0.8, 'should_checkpoint': True}"
+      ]
+     },
+     "execution_count": 9,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "best_result.config"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "8fac8cf8-67b6-4f84-9227-19db05957d5c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "best_result.log_dir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "88712bb6-2015-4417-966a-119caa31fb2a",
+   "metadata": {},
+   "source": [
+    "The `log_dir` will look something like: `PosixPath('~/ray_results/tune_analyzing_results/train_mnist_...')`\n",
+    "\n",
+    "The result `log_dir` gives the trial level directory that contains checkpoints (if you had checkpointing enabled) and logged metrics to load manually or inspect using a tool like Tensorboard (see `result.json`, `progress.csv`)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "3ec354ba-be67-444c-9433-1c5ebc8cbc40",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Get the last Ray AIR Checkpoint associated with the best-performing trial\n",
+    "best_result.checkpoint"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "a82b22d2-4e5b-46bb-8ce1-659f23a53c9f",
+   "metadata": {},
+   "source": [
+    "The `checkpoint` will look something like: `Checkpoint(local_path=.../checkpoint_00001)`."
```

Comment:
Ditto here: Instead of saying what it will look like, can we just run the `best_result.checkpoint` cell to actually show it?